src/lwc/bdiFieldMappings/bdiFieldMappings.js

```diff
@@ -247,13 +253,14 @@ export default class bdiFieldMappings extends LightningElement {
     * the currently selected object and child field mappings.
     */
     handleOpenModal() {
-        if (this.displayFieldMappings) {
-            fireEvent(this.pageRef, 'openModal', {
-                objectMapping: this.objectMapping,
-                row: undefined,
-                fieldMappings: this.fieldMappings
-            });
-        }
+        fireEvent(this.pageRef, 'openModal', {
```

Comment:
Do we not need the "if (this.displayFieldMappings)" checks any more? Wasn't there some event cross-contamination between the object and field modals?